bool all #turn on and all will be on. Turn off, each is individually enabled/disabled. Defaults to true.

bool surge #all axes can be individually changed. 
bool sway
bool heave
bool roll
bool pitch
bool yaw
